astropy/units/quantity.py

```diff
@@ -1696,27 +1696,32 @@ def itemset(self, *args):
         self.view(np.ndarray).itemset(*(args[:-1] + (self._to_own_unit(args[-1]),)))
 
     def tostring(self, order="C"):
+        """Not implemented, use ``.value.tostring()`` instead."""
```

Comment:
Hmm, it seems like there has been a change on how docstrings are inherited. How weird. Anyway, happy with this if it works!

Reply:
And not just happy if thjs works: this is actually an improvement, since now if someone does `quantity.tostring?` in ipython, they get a useful docstring!

Reply:
The problem was a Sphinx reference error with `frombuffer` from the docstring inherited from `ndarray.tobytes` :
```
❯ pydoc numpy.ndarray.tobytes 
...    
    See also
    --------
    frombuffer
        Inverse of this operation, construct a 1-dimensional array from Python
        bytes.
```
So adding our own docstring was the easy solution, and while doing that I tried to put a docstring that could be useful ;)